ecosystem/sep-TBD.md

```diff
@@ -0,0 +1,154 @@
+
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Interoperability Recommendations for NFTs
+Authors: Tomer Weller <tomer@stellar.org>, Frederic Rezeau <frederic.rezeau@gmail.com>
+Track: Informational
+Status: Draft
+Created: 02-22-2022
+Updated: 03-08-2022
+Version: 0.1.0
+Discussion: https://github.com/stellar/stellar-protocol/pull/1140
+```
+
+## Simple Summary
+This SEP provides informational guidelines on how NFTs can be managed on the Stellar network and within the ecosystem. It outlines some best practices on both _minting_ and _representing_ NFTs to maximize interoperability.
+
+
+## Dependencies
+The guidelines around NFT representation are an extension to [SEP-1][sep1-currency], specifically around the way currencies are represented.
+
+
+## Motivation
+With the NFT marketplace exploding in popularity, two needs arise: ecosystem _best practices_ for creating NFTs as well as _interoperability guidelines_ that ensure compatibility throughout the space. While both of these exist throughout the ecosystem, a single informational SEP is a necessary point of reference. Note that while the focus of this document is inspired by NFTs, it applies more broadly to NFTs that don't necessarily have strict adherence to non-fungibility.
+
+
+## Abstract
+We address two key concerns of non-fractional tokens: _minting_ (creation) and _representation_ (parsing, rendering, etc.). For the former, we outline a set of community-driven best practices (that are by no means a _required_ minting standard), and for the latter, we extend the [SEP-1 currency specification][sep1-currency] in order to maximize interoperability with how the ecosystem already renders assets.
+
+
+## Specification
+As already noted, this informational SEP is split into two parts: NFT _minting_ best practices and NFT _representation_ interoperability.
+
+The minting best practices are merely one way to create NFTs on Stellar and should not be taken as a standard. The interoperability guidelines, however, are important in representing your NFT via [SEP-1][sep1] and maximizing its compatibility with the greater ecosystem.
+
+### Minting NFTs
+This section presents a guide on _best practices_ (rather than a _standard_) for minting some types of NFTs. It's based heavily on [this community guide][litemint]. Your NFT may or may not need certain elements of this guide, or you may need additional components on top of it. These diversions are noted where appropriate.
+
+At a high level, your NFT is represented by an asset on the Stellar network. Owning the asset _represents_ owning the NFT. This makes your good a "first-class citizen" on the network: it immediately benefits from all of Stellar's native features for assets like [path payments](https://developers.stellar.org/docs/start/list-of-operations/#path-payment-strict-send) and the [decentralized exchange](https://developers.stellar.org/docs/glossary/decentralized-exchange/).
```

Comment:
> This makes your good a "first-class citizen"

Using good here instead of "asset" or just "it" is a bit confusing to me.

Reply:
Nice catch: the doc used to discuss "digital goods" generically but there were concerns about nomenclature, so the language was being amended to say NFT. This is a remnant of that change! Fixed in fe8c199.